### ISSUE TITLE
css `conic-gradient`

### ISSUE THREAD

**Issue by jsnkuhn** [web-platform-tests/interop#174]
### Description

From spec
> A gradient is an image that smoothly fades from one color to another. These are commonly used for subtle shading in background images, buttons, and many other things. The gradient functions described in this section allow an author to specify such an image in a terse syntax, so that the UA can generate the image automatically when rendering the page.

### Rationale

As I've been looking through the tests for other focus areas I randomly noticed that Safari seems to be failing most of the `conic-gradient` tests. I'm on windows so I checked caniuse (https://caniuse.com/?search=conic) a long time ago and assumed that conic-gradients were safe to use in all browsers. If support is really this bad in Safari I think there are a few projects that I'll have to go back and drop in fall back images for.

Can someone on Safari confirm that the conic-gradient tests are broken?

### Specification

https://drafts.csswg.org/css-images-4/#conic-gradients

### Tests

https://wpt.fyi/results/css/css-images?label=master&label=experimental&aligned&view=subtest&q=conic

**Comment by ydaniv** [comment]
From looking at the failing tests, these are not _really_ indicating lack of support, rather it looks like minor painting issues:

<img width="534" alt="image" src="https://user-images.githubusercontent.com/116658/193441985-a8501ba9-e52b-4217-9351-3ece9db39ac2.png">

A lot of pixels have values of RGB off by 1 from the expected result.

The "tiled" test has issues which are more apparent:

<img width="516" alt="image" src="https://user-images.githubusercontent.com/116658/193442087-8004c035-08c6-4613-9a62-4d94250c4e5e.png">

**Comment by foolip** [comment]
[Fuzzy matching](https://web-platform-tests.org/writing-tests/reftests.html#fuzzy-matching) could be used to make these tests pass, assuming some small deviations are expected.

**Comment by ydaniv** [comment]
> [Fuzzy matching](https://web-platform-tests.org/writing-tests/reftests.html#fuzzy-matching) could be used to make these tests pass, assuming some small deviations are expected.

I don't think we want to. At the end of the day these later become blockers for authors for releasing products over the web platform. These are probably best kept as failing, though not sure whether this also calls for an interop candidate 🤔

**Comment by foolip** [comment]
The question is how big the differences are, are they big enough to be observable? For [normalization-conic.html](https://wpt.fyi/results/css/css-images/normalization-conic.html?label=master&label=experimental&aligned&view=subtest&q=conic) the maximum pixel difference is just 1, and that's a case we often use fuzzy matching for.

Making the appropriate tests fuzzy also makes it clearer if there are any big differences left.

@jsnkuhn @ydaniv do you know of issues with conic gradients that aren't related to these failing tests? As a quick check I looked for "conic-gradient safari" on Stack Overflow, but didn't find anything.

**Comment by ydaniv** [comment]
Don't know of any.

> maximum pixel difference is just 1

The "tiled" test seems quite off on the edges, perhaps AA-related issues. The rest are off by 1 on each channel, perhaps less critical.

**Comment by jsnkuhn** [comment]
nope, i don't know of any other conic-gradient issues. 

Did find a webkit bug open for a bunch of failing `css-image` tests that does include the conic-gradient tests: https://bugs.webkit.org/show_bug.cgi?id=214456

**Comment by foolip** [comment]
In the State of CSS 2022 question about browser incompatibilities, there was only a single mention of `conic-gradient`, so not enough to make the top list in https://github.com/web-platform-tests/interop/issues/248.

The response was just "conic-gradients" with no further detail.
